tests/create_bucket_inventory/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "random_id" "name" {
+  byte_length = 6
+  prefix      = "tardigrade-s3-bucket-"
+}
+
+resource "aws_s3_bucket" "inventory" {
+  bucket = format("%s-%s", "inventory", random_id.name.hex)
+}
+
+/*moto has not implemented the bucket_inventory_configuration yet, so the CI test fails while running pytest.  The test passes when run against an actual AWS account.  We need to uncomment the folloiwng module when moto catches up.
```

Comment:
I feel like I would still prefer to exclude this in travis-ci, instead of commenting it out here. I think that would entail modifying the .travis-ci.yml to set:

```
      script: make mockstack/pytest PYTEST_ARGS="-k 'not create_bucket_inventory'"
```